process-engine/bpmn-studio
# 🚸 Always Display Overlays for CallActivities

**Changes:**

1. Always display overlays for CallActivities
2. Refactor LET
3. Fix checking if active elements changed
4. Fix checking if overlays exist
4. Add diagram preview modal

**Issues:**

Closes #1366

PR: #1370

## How can others test the changes?

- Open up the BPMN-Studio 
- Run a diagram with a CallActivity
- **Notice that if CallActivity is not active, there is a button for previewing the process model.**

## PR-Checklist

Please check the boxes in this list after submitting your PR:
